### ISSUE TITLE
`Help Wanted`: Akebi Failing to Close Mhyprot2

### ISSUE THREAD

**Issue by agentchromatic7** [Akebi-Private/Akebi-Issue-Tracker#267]
Good day, 

I'm not sure what's going on, but I played Genshin fine earlier and when I went to log back in its saying "Failed to close mhyprot2". I checked process hacker and mhyprot2 is not running. Why is it doing this and why are we getting this message. Is Akebi still safe to use?

![akebi](https://user-images.githubusercontent.com/51468103/225209008-6b073ef8-8e75-4321-9f27-5338bdc1ab11.png)

**Comment by ghost** [comment]
Did you remove `mhypbase.dll`?

**Comment by agentchromatic7** [comment]
> 
I have not deleted it. Should I reinstall genshin and akebi? Also, I just started Akebi up again and I didn't see mhypbase.dll or mhyprot2 running via process hacker.

**Comment by ghost** [comment]
try to close ph and run genshin

**Comment by agentchromatic7** [comment]
> try to close ph and run genshin

I closed process hunter and launched Genshin via Akebi. It's still saying it failed to close mhyprot2. I looked through task manager (details and services) and did not see mhyprot2 or mhypbase.dll running. I also ran Genshin without Akebi and did not see mhyprot2 or mhypbase.dll running.   

Is Akebi safe to use if mhyprot2 and mhypbase.dll are not running? I'm worried that they may suddenly start up and Akebi will be unable to block it. I'm not really sure how it works I'm just assuming that it could happen.

**Comment by ghost** [comment]
Did you tried to find `mhyprot2` handle (`Ctrl`+`F`) when you run game without cheat? If it doesn't exist, even if you run without cheat, then you don't need to worry about it. Because some CN users also said about this error.

**Comment by agentchromatic7** [comment]
> Did you tried to find `mhyprot2` handle (`Ctrl`+`F`) when you run game without cheat? If it doesn't exist, even if you run without cheat, then you don't need to worry about it. Because some CN users also said about this error.

Yes, I did search using ctrl + f. I did not find mhyprot2 running when starting Genshin without cheat, which is odd to me that its not there when using Genshin without cheat. However, if there is nothing to worry about then I will disregard it.

Just to confirm, as long mhyprot2 is not running when I start Genshin with Akebi cheat I should be in the clear, correct? Also, if by some chance mhyprot2 is running when I start Genshin using Akebi cheat, and Akebi fails to close mhyprot2, I can manual close mhyprot2 and continue using Akebi. Am I getting that correct?
